file-formats/clas/type/zif_aff_clas_v1.intf.abap

```diff
@@ -74,18 +74,22 @@ INTERFACE zif_aff_clas_v1 PUBLIC.
       "! <p class="shorttext">Message Class</p>
       "! The message class of the class
       message_class        TYPE c LENGTH 20,
-      "! <p class="shorttext">Type Descriptions</p>
-      "! Type descriptions
-      types                TYPE zif_oo_aff_types_v1=>ty_component_descriptions,
-      "! <p class="shorttext">Attribute Descriptions</p>
-      "! Attribute descriptions
-      attributes           TYPE zif_oo_aff_types_v1=>ty_component_descriptions,
-      "! <p class="shorttext">Event Descriptions</p>
-      "! Event descriptions
-      events               TYPE zif_oo_aff_types_v1=>ty_events,
-      "! <p class="shorttext">Method Descriptions</p>
-      "! Method descriptions
-      methods              TYPE zif_oo_aff_types_v1=>ty_methods,
+      "! <p class="sharttext"> Descriptions</p>
+      "! Descriptions maintained in SE80
+      BEGIN OF descriptions,
```

Comment:
I think we can put the declaration of the "descriptions" structure to 
https://github.com/SAP/abap-file-formats/blob/main/file-formats/typesUsedForClasAndIntf/zif_oo_aff_types_v1.intf.abap

and use refer to this structure in the intf and clas type

Reply:
@wurzka is preparing this one.